lib/trace_events.js

```diff
@@ -85,7 +84,7 @@ class Tracing {
 function createTracing(options) {
   validateObject(options, 'options');
 
-  if (!ArrayIsArray(options.categories)) {
+  if (!validateArray(options.categories, 'options.categories')) {
```

Comment:
The `validateArray()` validator does not return a boolean but rather throw the error directly if the condition is not met, the validator wasn't used intentionally to tell the user land that they must pass an array of strings as specified by the `string[]` type.